### DISCUSSION TITLE
Trigger Speed Test

### DISCUSSION THREAD

**Discussion by rac146** [Vaskivskyi/ha-asusrouter#838] [Q&A]
Is there a way to trigger a speed test run through this API? I can read speed test results, but don't see any way to trigger one unless I'm missing something..

**Comment by Vaskivskyi** [comment]
Currently, no. Please, follow #704 for updates. I will post there when the feature will be ready. And if it will be. There are some limitations on what can be done with the routers without a direct user interaction
